### PR TITLE
Fix trackingPrefix import

### PR DESCRIPTION
**Pull request by cehsu**
## Description
These changes fix a bug with how trackingPrefix was imported:

```
WARNING in ./src/applications/edu-benefits/feedback-tool/helpers.js
167:9-23 "export 'trackingPrefix' was not found in './config/form'
 @ ./src/applications/edu-benefits/feedback-tool/helpers.js
 @ ./src/applications/edu-benefits/feedback-tool/config/form.js
 @ ./src/applications/edu-benefits/feedback-tool/routes.jsx
 @ ./src/applications/edu-benefits/feedback-tool/feedback-tool-entry.jsx
```

## Definition of done
- [ ] ~Events are logged appropriately~
- [ ] ~Documentation has been updated, if applicable~
- [ ] ~A link has been provided to the originating GitHub issue (or connected to it via ZenHub)~
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/edu-benefits/feedback-tool/actions/schoolSearch.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import recordEvent from '../../../../platform/monitoring/record-event';
 
-import { trackingPrefix } from '../config/form';
```
Comment (rianfowler) [inline src/applications/edu-benefits/feedback-tool/actions/schoolSearch.js]: we really should prohibit default exports

**Review by rianfowler — APPROVED** [review]

**Review by erikphansen — COMMENTED** [review]
🙌 Thanks for fixing this. I just ran into it and tried to fix it, but I think the fix is more complicated than this (and Jenkins seems to agree)

**Review by erikphansen — APPROVED** [review]
My fix isn't working :(

File: src/applications/edu-benefits/feedback-tool/actions/schoolSearch.js (hunk 1 above)
Comment (cehsu) [reply src/applications/edu-benefits/feedback-tool/actions/schoolSearch.js]: what should be used in lieu?

**Pull request merged**